docs/pages/workflow/expo-cli.md

```diff
@@ -5,1083 +5,284 @@ maxHeadingDepth: 4
-| `--scheme [scheme]`                     | Custom URI protocol to use with a development build                                                            |
-| `-a, --android`                         | Opens your app in Expo Go on a connected Android device                                                        |
-| `-i, --ios`                             | Opens your app in Expo Go in a currently running iOS simulator on your computer                                |
-| `-w, --web`                             | Opens your app in a web browser                                                                                |
-| `-m, --host [mode]`                     | lan (default), tunnel, localhost. Type of host to use. "tunnel" allows you to view your link on other networks |
-| `--tunnel`                              | Same as --host tunnel                                                                                          |
-| `--lan`                                 | Same as --host lan                                                                                             |
-| `--localhost`                           | Same as --host localhost                                                                                       |
-| `--offline`                             | Allows this command to run while offline                                                                       |
-| `--dev`                                 | Deprecated: Dev mode is used by default                                                                        |
-| `--no-minify`                           | Deprecated: Minify is disabled by default                                                                      |
-| `--no-https`                            | Deprecated: https is disabled by default                                                                       |
-| `--config [file]`                       | Deprecated: Use app.config.js to switch config files instead.                                                  |
-
-</p>
-</details>
+- <kbd>a</kbd> Open the project in Expo Go on Android.
+- <kbd>shift+a</kbd> Select an Android device or emulator to open.
 
----
+- <kbd>i</kbd> Open the project in Expo Go on iOS.
+- <kbd>shift+i</kbd> Select an iOS Simulator to open.
+- <kbd>w</kbd> Open the project in a web browser. This may require Webpack to be installed in your project.
 
-### Auth
+- <kbd>r</kbd> Reload the app on any connected device.
+- <kbd>m</kbd> Open the dev menu on any connected native device (web not support).
+- <kbd>shift+m</kbd> Choose more commands to trigger on connected devices. This includes toggling the performance monitor, opening the element inspector, reloading the device, and opening the dev menu.
+- <kbd>j</kbd> Open Chrome Dev Tools for any connected device that is using Hermes as the JavaScript engine. [Learn more](/guides/using-hermes/).
```

Comment:
The link here is not showing when running the docs locally:
![CleanShot 2022-08-15 at 19 35 39@2x](https://user-images.githubusercontent.com/10234615/184650603-7e55beef-8e54-4a13-9ca3-e8de64cc5b38.png)

But works fine if placed outside the bullet point.

Also, I think we should be explicit about when mentioning the links such as: "For more information, see `[Using Hermes](link-to-the-docs)`".  As to, where the link is going to lead to the docs reader.

Reply:
Also the inline code block are not rendered properly. Probably MDX treat the whole sentence as HTML since we use the `kbd` tag at the beginning. Not sure if there is an easy way to fix this.